tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -224,7 +224,8 @@ Bridge network - vCenter none
     Run Keyword And Ignore Error  Cleanup Datastore On Test Server
 
     ${output}=  Run  bin/vic-machine-linux create --name=%{VCH-NAME} --target="%{TEST_USERNAME}:%{TEST_PASSWORD}@%{TEST_URL}" --thumbprint=%{TEST_THUMBPRINT} --image-store=%{TEST_DATASTORE} ${vicmachinetls}
-    Should Contain  ${output}  FAILURE
+    Should Contain  ${output}  ERROR
+    Should Contain  ${output}  An existing distributed port group must be specified for bridge network on vCenter
```

Comment:
the vcenter here still has me concerned, does this work on ESX?  does it still report this message?  if so, we need a bug to fix the output, but either way you should cut the check off right before that in case it gets fixed.